openassessment/assessment/test/test_peer.py

```diff
@@ -218,7 +218,7 @@ def test_get_waiting_step_details(self):
         self._create_student_and_submission("Tim", "Tim's answer")
         bob_sub, bob = self._create_student_and_submission("Bob", "Bob's answer")
         peer_api.get_submission_to_assess(bob_sub['uuid'], 1)
-        assessment = peer_api.create_assessment(
```

Comment:
this and `test_flexible_peer_grading__zero_graded` changes were quality issues that have somehow slipped in